src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -227,12 +279,14 @@ export class SearchMenu extends React.Component {
                       aria-selected={JSON.stringify(
                         selectedItem === suggestion,
                       )}
+                      data-index={index}
```

Comment:
Is this used? 🙂

Reply:
good catch!

Reply:
/ where is the shame emoji when you need it